pkg/virtctl/imageupload/imageupload.go

```diff
@@ -307,6 +307,7 @@ func getHTTPClient(insecure bool) *http.Client {
 	client := &http.Client{}
 
 	if insecure {
+		// #nosec
```

Comment:
We should add an explanation what issue has been detected and why it is ok to skip the security check here.

Same goes for the other nosec markers.

Reply:
Good point.